client/src/TenderList.js

```diff
@@ -50,6 +60,10 @@ const TendersList = () => {
 		}
 	};
 
+	const hasSubmittedBid = (tenderId) => {
+		return bids.some((bid) => bid.tender_id === tenderId);
+	};
```

Comment:
This will not work on all scenarios because `/api/bidder-bid` is paginated. Better to have `/api/tenders` response to include the bids for the returned tenders